usr.bin/man/man.1

```diff
@@ -1,27 +1,29 @@
 .\"-
-.\"  Copyright (c) 2010 Gordon Tetlow
-.\"  All rights reserved.
+.\" SPDX-License-Identifier: BSD-2-Clause
```

Comment:
one more space would put it in column with the removed lines. pretty useless nit so feel free to ignore

Reply:
Neat and consistent is meaningful! Unfortunately, I can't on this one because the SPDX spec defines that.

Reply:
Remove one leading space in the lines above to make it consistent.

Reply:
When I get home I think I'll revert tagging spdx, since it isn't really helping anything.